src/main/kotlin/com/pubnub/api/retry/RetryableEndpointGroup.kt

```diff
@@ -1,14 +1,49 @@
 package com.pubnub.api.retry
 
+import com.pubnub.api.PubNub
+import com.pubnub.api.retry.RetryableEndpointGroup.ACCESS_MANAGER
+import com.pubnub.api.retry.RetryableEndpointGroup.APP_CONTEXT
+import com.pubnub.api.retry.RetryableEndpointGroup.CHANNEL_GROUP
+import com.pubnub.api.retry.RetryableEndpointGroup.FILE_PERSISTENCE
+import com.pubnub.api.retry.RetryableEndpointGroup.MESSAGE_PERSISTENCE
+import com.pubnub.api.retry.RetryableEndpointGroup.MESSAGE_REACTION
+import com.pubnub.api.retry.RetryableEndpointGroup.PRESENCE
+import com.pubnub.api.retry.RetryableEndpointGroup.PUBLISH
+import com.pubnub.api.retry.RetryableEndpointGroup.PUSH_NOTIFICATION
+import com.pubnub.api.retry.RetryableEndpointGroup.SUBSCRIBE
+
+/**
+ * Enum representing various retryable endpoint groups.
+ *
+ * @property SUBSCRIBE Group for [PubNub.subscribe] operations.
```

Comment:
These won't show up on the enum values themselves, the proper way to document them would be like this:
```
enum class MyEnum {
/**
* This enum does blah
*/
ENUM_VALUE_1,

/**
* This other enum does blah
*/
ENUM_VALUE_2,
}
```

Reply:
I see. I will change this.